CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-﻿# 0.8.2
+# 0.8.3
+* Results example type: sort after lookups
```

Comment:
If I'm reading the diff correctly, you also made it properly respect `include`, right?

Reply:
Done.